src/generic.lisp

```diff
@@ -134,3 +134,9 @@ Force a reload & free old memory when FORCE-RELOAD is T."))
 
 (defgeneric image-draw (image obj)
   (:documentation "Draw an object OBJ onto the the rl-image IMAGE."))
+
+(defgeneric setup-scene (scene)
```

Comment:
Pedantic quibble: "Setup" is a noun; "set up" is a verb. To match the other one and be grammatically correct, we want the verb, i.e. `set-up-scene`.

Reply:
Funnily, I consulted a dictionary for this because I was trying to minimize symbol length and still got it wrong :man_facepalming:, fixing...